generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -33,6 +33,7 @@ import {
 } from "./EndpointRequest";
 import { CodeBlock } from "@fern-api/generator-commons";
 import { assertNever } from "@fern-api/core-utils";
+import { FileUploadBodyProperty } from "@fern-fern/ir-sdk/serialization";
```

Comment:
I think we'll want to use `@fern-fern/ir-sdk/api` here, but I'm sure this works fine as-is because they share a common interface.